3rdparty/quickjs/quickjs.c

```diff
@@ -34691,6 +34691,26 @@ int JS_SetModuleExportList(JSContext *ctx, JSModuleDef *m,
     return 0;
 }
 
+// https://www.freelists.org/post/quickjs-devel/Patch-Add-some-informations-to-get-more-informations-from-compiled-modules
+int JS_GetModuleExportEntriesCount(JSModuleDef *m)
+{
+    return m->export_entries_count;
+}
+
+JSValue JS_GetModuleExportEntry(JSContext *ctx, JSModuleDef *m, int idx)
+{
+    if (idx >= m->export_entries_count || idx < 0)
+        return JS_UNDEFINED;
+    return JS_DupValue(ctx, m->export_entries[idx].u.local.var_ref->value);
+}
+
+JSAtom JS_GetModuleExportEntryName(JSContext *ctx, JSModuleDef *m, int idx)
+{
+    if (idx >= m->export_entries_count || idx < 0)
+        return JS_ATOM_NULL;
+    return JS_DupAtom(ctx, m->export_entries[idx].export_name);
+}
+
```

Comment:
I think we could just #include "../quickjs_patch/quickjs.c" here, and keep 3rdparty/quickjs_path with quickjs.c and quickjs.h (containing the extra signatures below).

It's not as generic as the patch, but it's a single, can't-go-out-sync copy of the changes, with a one-line, trivial-to-merge update to upstream. I'd prefer it, but I'm also happy to go with this is we have consensus.

Reply:
Addressed in #1472.